recipes/pymirc/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pymirc" %}
+{% set version = "0.25" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymirc-{{ version }}.tar.gz
+  sha256: 0ec860399f222af6e4880a8c86f02a380a932058b572d8cccd3ceef7094e8a84
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.9
+    - setuptools-scm
+    - setuptools_scm_git_archive
+  run:
+    - matplotlib-base >=2.2.2
+    - nibabel >=3.0
+    - numba >=0.39
+    - numpy >=1.15
+    - pydicom >=1.1
+    - python >=3.6,<3.9
+    - scikit-image >=0.14
+    - scipy >=1.1
+
+test:
+  imports:
+    - pymirc
+    - pymirc.fileio
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/gschramm/pymirc
+  summary: Python imaging utilities developed in the medical imaging research center of KU Leuven
+  license: LGPL-3.0
```

This should end with `-only` or `-or-later`. As the source repo doesn't appear to contain "or any later version" I guess its:
```suggestion
  license: LGPL-3.0-only
```